src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFVersionViewController.swift

```diff
@@ -121,7 +121,8 @@ class HealthCertificatePDFVersionViewController: DynamicTableViewController, UIA
 		var pdfFileName = "healthCertificates.pdf"
 		
 		if !viewModel.certificatePersonName.isEmpty {
-			pdfFileName = "healthCertificate_\(viewModel.certificatePersonName).pdf"
+			let displaceName = viewModel.certificatePersonName.replacingOccurrences(of: " ", with: "_")
+			pdfFileName = "healthCertificate_\(displaceName).pdf"
```

Comment:
It's technical okay, but from MVVM it would be nice when those informations like filename and display name comes from VM.